src/app/components/team/TeamComponent.js

```diff
@@ -288,11 +288,11 @@ class TeamComponent extends Component {
           { tab === 'edit'
             ? <TeamDetails team={team} />
             : null }
-          { isSettings && tab === 'lists'
+          { isSettings && tab === 'columns'
             ? <TeamLists key={tab} />
             : null }
-          { isSettings && tab === 'metadata'
-            ? <TeamTasks key={tab} team={team} fieldset="metadata" />
+          { isSettings && tab === 'annotation'
+            ? <TeamTasks key={tab} team={team} fieldset="annotation" />
```

Comment:
The value for `fieldset` remains `metadata` for now. This is a backend property that hasn't been updated yet.